inc/register-settings.php

```diff
@@ -1,4 +1,4 @@
 <?php
 
 // Menu registration
-register_nav_menu("primary", "Navigation");
+register_nav_menu("Home", "Services", "Volunteer", "About");
```

Comment:
We actually don't need to change this line from what it was. You can check the documentation for register_nav_menu for the proper usage if curious.